### PR TITLE
Clean up

### LINKED ISSUE #4: Fix Twilio rejection

### LINKED ISSUE #6: Remove APIs that will not be used (Twilio, Delivery time)

### PR DESCRIPTION
**Pull request by noobjey**
closes #4 
closes #6
General cleanup of gems and features we are not going to use.
- Remove twillio
- Remove google directions
- Remove skylight

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -1,12 +1,10 @@
-require 'twilio-ruby'
-
 class OrdersController < ApplicationController
 
   def create
     if current_user
       create_order
       empty_cart
-      notify_user
+      flash[:success] = "Order placed! Dinners on the way!"
```
Comment (MowAlon) [inline app/controllers/orders_controller.rb:7]: Typo, yo, typo. Dinners = Dinner's

File: app/controllers/orders_controller.rb [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
       redirect_to orders_path
     else
       flash[:warning] = "Sign In to complete your order, Dinners almost ready!"
```
Comment (MowAlon) [inline app/controllers/orders_controller.rb:10]: Another typo to catch... Sign In = Sign in

**Comment by MowAlon** [comment]
Merging, despite typos

File: app/controllers/orders_controller.rb (hunk 1 above)
Comment (noobjey) [inline app/controllers/orders_controller.rb:7]: we can get rid of all this later, when we rebrand

**Pull request merged**